src/stack.js

```diff
@@ -1,5 +1,49 @@
-'use strict'
```

Comment:
@jason00111 also 'use strict' is not necessary so it only made it easier to get rid of it, since the spread operator is so useful